### ISSUE TITLE
Error "/home/samuel/qt/gf2-gui/serial.cpp:32: error: expected type-specifier before ‘QRegExpValidator’

### ISSUE THREAD

**Issue by samuelfmlourenco** [bloguetronica/gf2-gui#2]
The error shown in the title happens when compiling using Qt 5.15.3. The full error is shown below:

```
/home/samuel/qt/gf2-gui/serial.cpp:32: error: expected type-specifier before ‘QRegExpValidator’
../gf2-gui/serial.cpp: In constructor ‘Serial::Serial(QWidget*)’:
../gf2-gui/serial.cpp:32:42: error: expected type-specifier before ‘QRegExpValidator’
   32 |     ui->lineEditSerial->setValidator(new QRegExpValidator(QRegExp("[A-Za-z\\d-]{1,12}"), this));
      |                                          ^~~~~~~~~~~~~~~~
```

**Comment by samuelfmlourenco** [comment]
This can be fixed by including the QRegExpValidator library in "serial.cpp". However, QRegExp should be included as well, as follows:

```
#include <QRegExp>
#include <QRegExpValidator>
```

**Comment by samuelfmlourenco** [comment]
Fixed in version 2.1.
